src/System.Runtime.InteropServices/System.Runtime.InteropServices.sln

```diff
@@ -19,28 +31,68 @@ Global
 		Windows_Release|Any CPU = Windows_Release|Any CPU
 	EndGlobalSection
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
-		{EC6AA4D9-B3E8-4CCA-8AB1-8BBFD89266AE}.Debug|Any CPU.ActiveCfg = Windows_Debug|Any CPU
-		{EC6AA4D9-B3E8-4CCA-8AB1-8BBFD89266AE}.Debug|Any CPU.Build.0 = Windows_Debug|Any CPU
+		{EC6AA4D9-B3E8-4CCA-8AB1-8BBFD89266AE}.Debug|Any CPU.ActiveCfg = net46_Debug|Any CPU
```

Comment:
I guess VS defaults to the alphabetically-first configuration. I'm just going to leave it, I think we will probably auto-generate all these in the future and we can just set what we want the default to be then.